src/features/comments/comment.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from "react";
+import { CommentsList } from "./commentsList";
+import {
+  Divider,
+  Box,
+  Typography,
+  IconButton,
+  List,
+  ListItemText,
+} from "@mui/material";
+import ForumRoundedIcon from "@mui/icons-material/ForumRounded";
+import NorthIcon from "@mui/icons-material/North";
+import SouthIcon from "@mui/icons-material/South";
+import ReplyIcon from "@mui/icons-material/Reply";
+import { relativeDays } from "../../components/helperFunctions";
+export const Comment = ({ comment }) => {
+  const replies = comment.data?.replies?.data?.children;
+  const date = new Date(comment.data.created_utc * 1000);
+  const [showReplies, setShowReplies] = useState(false);
+  const toddleReplies = () => {
+    const newShowReplies = !showReplies;
+    setShowReplies(newShowReplies);
+  };
+
+  return (
+    <List sx={{ borderLeft: "0.15rem inset #edeff1" }}>
+      <ListItemText
+        primary={
+          comment.data.body && (
+            <Box
+              sx={{
+                cursor: replies ? "pointer" : "auto",
+                p: "0 1rem",
+                width: "fit-content",
+              }}
+              onClick={toddleReplies}
+            >
+              <Box sx={{ width: "max-content" }}>
+                <Typography variant="h7" gutterBottom={true}>
```

Comment:
I am spotting a lot of variant=h7, probably worth creating a typography component for that and name it for what variant=h7 stands for in your GUI